wqflask/tests/unit/wqflask/marker_regression/test_rqtl_mapping.py

```diff
@@ -1,42 +1,47 @@
 import unittest
 from unittest import mock
-from wqflask import app
-from wqflask.marker_regression.rqtl_mapping import get_trait_data_type
-from wqflask.marker_regression.rqtl_mapping import sanitize_rqtl_phenotype
-from wqflask.marker_regression.rqtl_mapping import sanitize_rqtl_names
+from wqflask.marker_regression.rqtl_mapping import run_rqtl
 
+class AttributeSetter:
```

Comment:
*nitpick: comment:* In mocks, I normally prefer using dataclasses. They tend to be easier to read, thereby easier to maintain...